README.md

```diff
@@ -1,2 +1,74 @@
 # Ruby-Tic-Tac-Toe
-Tic-Tac-Toe Game 
\ No newline at end of file
+ Tic-Tac-Toe is a game for two players, X and O, who take turns marking the spaces in a 3×3 grid. The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row is the winner.
+## Built With
+
+- Ruby
+
+## Getting Started
+
+## Instructions to play the Game
+
+- Two players are required to play the game.
+- The first player is X and the second player is O.
+- The players are required to input numbers between 0 - 8, to pick a position in the board.
+- If the player pick a position that is occupied, the player will be required to pick another position.
+- The first player to get identical letter in the row or column or left and right diagonal wins the game.
+- If no identical letter occurs and the board is full, the game will end in a draw
+
+## How to run the game in your terminal
+
+- Run this command line: bin/main.rb
+
+**This is an example of how you may give instructions on setting up your project locally.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Install ruby on your local machine
+
+### Setup
+
+- Clone the repository on your local machine
+- cd into the folder, find the bubble_sort file and open it
+
+### Install
+
+- Install Ruby (most recents versions)
+- Install VSCode or any code editor you like
+- Add gem 'rubocop' to Gemfile (not sure how to use Gemfile? Read <a href="https://bundler.io/v1.15/guides/bundler_setup.html">this</a>).
+- Run bundle install.
```

Comment:
- [ ] Final users don't need VSCode, or rubocop they only need to have ruby installed in their computers so let's put these instructions in a section for developers or contributors